account_analytic_sequence/data/sequence.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
```

Comment:
do we have to put license file in the views?

Reply:
There's no requirement on that, yet this file is still some sort of code, and should contain license/copyright notice